Repeat/repeat.py

```diff
@@ -0,0 +1,56 @@
+
+from PIL import Image
+import numpy as np     # numpy
+from collections import namedtuple
+
+repeat = namedtuple('repeat', 'count pixel')
```

Comment:
It should be
```python
namedtuple('repeat', ['count', 'pixel'])
```
https://docs.python.org/2/library/collections.html#collections.namedtuple